nomt/src/bitbox/beatree/writeout.rs

```diff
@@ -379,7 +379,7 @@ impl MetaSwap {
         if let Some(new_meta) = self.new_meta.take() {
             // Oh god, there is a special place in hell for this. Will do for now though.
             let mut page = Box::new(Page::zeroed());
-            new_meta.encode_to(&mut page.as_mut()[..24].try_into().unwrap());
+            new_meta.encode_to(&mut page.as_mut()[..20].try_into().unwrap());
```

Comment:
I just notice, looking at the `Meta` struct that it doesn't even need to be 20 bytes long, but only 16. I will update the code in meta.rs and also here in a follow up pr!